### ISSUE TITLE
TWE: Create Research Plan for RP[###]

### ISSUE THREAD

**Issue by lkawentel** [hackforla/internship#320]
### Dependencies
- [ ] [Replace this with Roadmap issue #]
- [ ] A research setup structure (wiki, GitHub labels, Google Drive, etc.) has been created and approved.

### Overview
We need to create a Research Plan for [NAME OF RESEARCH PLAN] so that [RESEARCH GOAL(S)].

### Action Items
- [ ] Update this issue to make sure it's properly categorized and easy to manage
     - [ ] Add to the Project Board (helps with Project Management overview)
     - [ ] Add the milestone: [Research Plan Creation](https://github.com/hackforla/internship/milestone/29) (helps with prioritization)
     - [ ] Add label of the correct research plan number associated with this research plan (e.g., Research: RP001) (helps you find related issues in the same workflow)
     - [ ] Edit title of issue with the correct research plan number (same as prior)
     - [ ] Edit placeholders in Overview of this issue with the relevant information in square brackets. (personalizes the issue to make it easier to read). Any place you see [replace...] replace both the text and the brackets with what it is asking for (including in the front matter above but not where it says name, but not where it says title.
- [ ] Update this issue with the relevant Resources:
     - [ ] Go to the Research Output Overview Page (link in Resources) and find the relevant overview type wiki page. Then, change `[Wiki [TYPE] Research Overview Page]` in the Resources below with the correct wiki page of the research overview by type.
     - [ ] Go to the Research Output Overview Page again and find the relevant wiki page for this specific research plan. Then, change the `[RP__ Wiki Page]` in the Resources below with this RP's specific wiki page.
     - [ ] Find and link the folder you are going to put your research plan in  (e.g., Research by Type>2. Intern>RP6 - Intern Intake Interviews)
        - [ ] Go to the Google Drive's Research by Type Folder (link in Resources)
        - [ ] Go into the relevant Type Folder (e.g., 1. Mentor, 2. Intern, etc.) 
        - [ ] once inside, right-click on the relevant destination folder and copy the link. (e.g., RP6 - Intern Intake Interviews)
         - [ ] Add the Google Drive link in the Resources below with the link you just copied and revise the display text to include the Research Plan number (e.g., `Google Drive RP__ Folder` to `Google Drive RP008 Folder`)
- [ ] Create the Research Plan Document.
     - [ ] Make a new document using the TWE Research Plan template (link in Resources). Follow the instructions in the How To Write a Research Plan guide (link in Resources) and mark your progress in this issue.
     - [ ] Move your document from "My Drive" to the shared drive internship folder of this specific RP.
     - [ ] Name your document to be called "TWE: RP___ : [TITLE OF RESEARCH] Research Plan".
     - [ ] Update the document with the relevant information in accordance with the research roadmap document (found on the Research Overview Page).
     - [ ] Update the Table of contents (page 2) when you are finished with the document.
- [ ] Meet and sign-off research plan document with UX Research lead (by adding it to the research team meeting agenda).
- [ ] Update RP Wiki Page with any changes or corrections.
- [ ] Once research lead has signed off, add it to the PM/Research lead agenda
- [ ] sign-off by PM 
- [ ] Does this issue have any dependency checkboxes in the action items here?
   - [ ] if yes, remove dependencies that are listed below (issues that are waiting on this issue to be completed.  Sometimes they have already been made and sometimes they yet to be made).
       - [ ] close this issue
   - [ ] if no, check to see if the follow-up issues have already been made (see related structure issue).  If new issues have not been made, then apply label `ready for research lead` and define what needs to be done in a comment below
      - [ ] close the issue
      - [ ] move to `Question/Review` Column

### Resources
- [Wiki Research Output Overview Page](https://github.com/hackforla/internship/wiki/Research-Output-Overview)
- [Wiki [TYPE] Research Overview Page]
- [RP__ Wiki Page]
- [Google Drive Research by Type Folder](https://drive.google.com/drive/folders/1f5Qgq-ikT_UwcgRBuoBamqY0Wacsg9f5?usp=sharing)
- [Google Drive RP__ Folder]
- [Research Plan Template](https://docs.google.com/document/d/1SDW6n086PnSrrVSmPgQZWhlkz5_yC70vDvp7Lx7u0Wo/template/preview)
- [TWE: How to Write a Research Plan](https://docs.google.com/document/d/1Cwc0w4ZPUI8989w3jU8BW2LzLK_Tl5gHdI0VxN5ej0o/edit)

**Comment by bradyse** [comment]
@lkawentel 

Please provide update
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."

**Comment by sharongupta1** [comment]
This issue does not have the name of the research plan and can't be prioritized until it does
@bradyse @lkawentel

**Comment by bradyse** [comment]
We updated the content of the issue and unassigned Linda so that she can work on an analysis demo for rapid research to be given at our next UXR team meeting.

**Comment by dialectic51** [comment]
Spoke to Bonnie at All Team Meeting, where she made the following points with regard to the concerns I raised in [my previous comment:](https://github.com/hackforla/internship/issues/320#issuecomment-1433410916)

- Sending out a survey won't necessarily give a better response rate. In fact, in the past, requesting an interview has resulted in a much higher response rate (people value the human interaction).
  - Therefore, plan to conduct semi-structured interviews as noted in the [RP020 research plan](https://docs.google.com/document/d/1GK3jaUlJerLtUFZhIXXAM2oDZDeYBWyooPXH0Z3IFhw/edit) contained in the [google drive.](https://drive.google.com/drive/folders/1Mt099BzsWqhBe-y02ReDIEsRHB7vNdWR)
-  We have a [spreadsheet in the google drive](https://docs.google.com/spreadsheets/d/1Y15zT64U4fPu1QXSQJWwRg_DZZL29wtJZqkLpcdvdCU/edit#gid=819048381) that lists all of the pull requests and other interactions between HfLA volunteers and interns (both working on the website).
- Need to craft the set up/introductory message for volunteers (i.e., request for participation) and the interview questions
  - During the interview, the interviewer will share the information from the spreadsheet (on intern-volunteer interactions) with the volunteers, and then have the volunteers use the "talk aloud" method as they review their pull requests and other interactions with the interns (i.e., say out loud the thoughts they have about the intern and their interactions with them).
  - Finally, ask the interview questions.

**Comment by dialectic51** [comment]
Progress: I have started revising [RP020,](https://docs.google.com/document/d/1GK3jaUlJerLtUFZhIXXAM2oDZDeYBWyooPXH0Z3IFhw/edit#) making detailed comments to the document (where questions exist) and filling out the unfinished portion of the RP. I identified KPI's for each of the research objectives, put together a proposed timeline for completing the research, and added 3 appendices. Appendix I is the script for recruiting participants (complete draft). Appendix II is an outline of the interview protocol. It includes Likert-scale questions for measuring KPI's, but the interview questions still need to be written. Appendix III will be the consent form. (I am waiting for the consent form template to be completed, which I will then use to create the text for the consent form).
Blockers: I will need the lead UXR and Bonnie to review my contributions and suggest changes. I'm waiting on the creation of the consent form template, but if its creation is delayed, I can go ahead and write up a consent form so that we can start the recruitment process. 
Availability: 6 hours
ETA: I should be able to write out some of the actual interview questions next week prior to Tuesday's UXR meeting.

**Comment by dialectic51** [comment]
**Progress:** Attempted to fix the Objectives and KPI's based on Bonnie's feedback; not successful.
- (Note: Objectives and KPI's should be based on the research roadmap, not the research objectives and questions listed in RP020).

**Blockers:** 
- 1.) Unclear what the purpose of the "Goals and Metrics" section of the RP020 is, which contains the Objectives and KPIs. 
  - 1a.) How will the information listed in this section impact or shape the interviews we will collect?
  - 1b.) Can we proceed with conducting the interviews (time sensitive) before we figure out the KPIs?
- 2.) [TWE Research Roadmap: Other HfLA Team Research](https://github.com/hackforla/internship/issues/328) doesn't provide enough information to create Objectives or KPIs.
  - 2a.) There is no mention of Objectives or KPIs in the Research Roadmap
- 3.) Should the quantitative metrics listed in the [wiki for Research Plan 20: Intern Interaction with HfLA Teams](https://github.com/hackforla/internship/wiki/Research-Plan-20:-Intern-Interaction-with-HfLA-Teams) be included in the Research Plan for RP020?
  - 3a.) Does the quantitative data need to be collected or analyzed before we conduct the interviews?  
  - 3b.) Currently, RP020 only includes qualitative methods that are "based upon the quantitative data collected."
  - 3c.) If the collection and analysis of the quantitative metrics are not part of RP020, are KPI's necessary?
- To sum up, the Objectives and KPIs in the Goals and Metrics section appear to be an unnecessary impediment to moving forward with conducting the interviews.

**Availability:** 6 hours

**ETA:** Depends upon when the questions listed under the blockers gets clarified.

**Comment by dialectic51** [comment]
**Progress:**
- I have been working with Ying to fill out the [TWE: Research Roadmap: HfLA Volunteers.](https://docs.google.com/document/d/1fVOsbi4yZqxMMguIsC3dOFxRkpoBImNbDCG53wxLn7Q/edit#) 
- I have written out a draft of the interview protocol, including interview questions, for [TWE: Research Plan: RP020](https://docs.google.com/document/d/1GK3jaUlJerLtUFZhIXXAM2oDZDeYBWyooPXH0Z3IFhw/edit#)

Blockers
- The TWE: Research Roadmap: HfLA Volunteers document needs to be completed before the TWE: Research Plan: RP020 can be completed.
  - The TWE: Research Roadmap: HfLA Volunteers document needs to be reviewed by Bonnie
- Still unclear what the purpose of the "Goals and Metrics section of RP020 is. [See questions 1 and 2 in my comment above.](https://github.com/hackforla/internship/issues/320#issuecomment-1448686285)
- Should also clarify the relationship of the quantitative metrics we plan to collect to RP20. [See question 3 in my comment above](https://github.com/hackforla/internship/issues/320#issuecomment-1448686285)

**Availability:** 6 hours

**ETA:** Hoping to have Roadmap and RP20 finalized by next week.
